routes/departments.go

```diff
@@ -46,3 +47,38 @@ func GetDepartments(ctx *gin.Context) {
 
 	ctx.JSON(http.StatusOK, resp)
 }
+
+// DeleteDepartment godoc
+// @Summary    delete a department
+// @Tags       departments
+// @Produce    json
+// @Param      id                              path        uuid                    true    "department id"
```

Comment:
for openapi it's just a string (also see https://github.com/helpwave/rest-api/blob/main/routes/rooms.go#L23)

Reply:
```suggestion
// @Param      id                              path        string                    true    "department id"
```